proofs/src/plonk/circuit.rs

```diff
@@ -97,6 +97,18 @@ impl<C: ColumnType> PartialOrd for Column<C> {
     }
 }
 
+#[cfg(feature = "extraction")]
+pub mod extraction {
+    //! Extraction support for column types.
+
+    use extractor_support::{auto_conf_impl, circuit::configuration::AutoConfigure};
+
+    auto_conf_impl!(super::Column<super::Fixed>, fixed_column, crate);
+    auto_conf_impl!(super::Column<super::Instance>, instance_column, crate);
+    auto_conf_impl!(super::Column<super::Advice>, advice_column, crate);
+    auto_conf_impl!(super::TableColumn, lookup_table_column, crate);
```

Comment:
This trait is used by the extractor while working with the chip's configuration. Allows for passing a `&mut ConstraintSystem<F>` between complex types and needs leaf implementations for these types.

- [Docs for the trait](https://docs.rs/mdnt-support/latest/mdnt_support/circuit/configuration/trait.AutoConfigure.html)
- [Docs for the macro](https://docs.rs/mdnt-support/latest/src/mdnt_support/circuit/configuration.rs.html#14-25)